Tests/scripts/infrastructure_tests/release_notes_generator_test.py

```diff
@@ -655,3 +655,63 @@ def test_merge_rns_with_gerneral_announcment(self, Pack_name, versions_ls, expec
         rn_block, latest_version = merge_version_blocks(pack_versions_dict)
         assert latest_version == expected_version
         assert rn_block == expected_results
+
+    @pytest.mark.parametrize('pack_rns, expected_rns, expected_version', [
+        pytest.param({
+            "1.0.1": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Updated the Docker image to: 123.",
+            "1.0.2": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Updated the Docker image to: 456.",
+        },
+            "#### Integrations\n"
+            "##### Some Integration\n"
+            "- Updated the Docker image to: 456.",
+            "1.0.2", id="Merge docker updates"),
+        pytest.param({
+            "1.0.1": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Added the new update to the integration.\n"
+                     "- Updated the Docker image to: 123.",
+            "1.0.2": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Added the other update to the integration.\n"
+                     "- Updated the Docker image to: 456.",
+        },
+            "#### Integrations\n"
+            "##### Some Integration\n"
+            "- Added the new update to the integration.\n"
+            "- Added the other update to the integration.\n"
+            "- Updated the Docker image to: 456.",
+            "1.0.2", id="Merge docker updates with other updates"),
+        pytest.param({
+            "1.0.1": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Added the new update to the integration.\n"
+                     "- Updated the Docker image to: 123.",
+            "1.0.2": "#### Integrations\n"
+                     "##### Some Integration\n"
+                     "- Added the other update to the integration.\n",
+        },
+            "#### Integrations\n"
+            "##### Some Integration\n"
+            "- Added the new update to the integration.\n"
+            "- Added the other update to the integration.\n"
+            "- Updated the Docker image to: 123.",
+            "1.0.2", id="Handle old docker update"),
+    ])
+    def test_merge_rns_with_several_docker_updates(self, pack_rns: dict, expected_rns: str, expected_version: str):
+        """
+        Given: Two consecutive versions of RNs.
```

Comment:
Let's add a case of two DIFFERENT docker images for example:
demisto/imageX and demisto/imageY where each has several updates and verify that after squashing we see only one message for each image.

Reply:
Not sure I understood your intention, is the test case I added what you were looking for?